pkgs/applications/audio/puddletag/default.nix

```diff
@@ -43,6 +39,11 @@ python3.pkgs.buildPythonApplication rec {
       --replace share/pixmaps share/icons
   '';
 
+  buildInputs = [
+    qtbase
+    qtwayland
+  ];
```

Comment:
Why do we need these now? I think we should have always included `qtbase`, but `qtwayland`?